ape_aws/kms/_cli.py

```diff
@@ -54,7 +63,103 @@ def create_key(
     cli_ctx.logger.success(f"Key created successfully with ID: {key_id}")
 
 
-# TODO: Add `ape aws kms import`
+@kms.command(name="import")
+@ape_cli_context()
+@click.option(
+    "-p",
+    "--private-key",
+    "private_key_path",
+    type=click.Path(),
+    help="The private key you intend to import",
+    metavar="Path",
+)
+@click.option(
+    "-a",
+    "--admin",
+    "administrators",
+    multiple=True,
+    help="Apply key policy to a list of administrators if applicable, ex. -a ARN1, -a ARN2",
+    metavar="list[ARN]",
```

Comment:
This makes sense as a metavar, but it should display like this `-a ARN -a ARN ...`
```suggestion
    metavar="ARN",
```